ui/src/components/executions/FilePreview.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <csv v-if="extensionToMonacoLang === 'csv'" :value="formattedContent" />
+    <img v-else-if="extensionToMonacoLang === 'image'" :src="formattedContent" alt="Image output preview">
+    <editor v-else :model-value="formattedContent" :lang="extensionToMonacoLang" format-on-start />
+</template>
+<script>
+    import Editor from "../inputs/Editor.vue";
+    import * as ion from "ion-js";
+    import Papa from "papaparse";
+    import Csv from "../Csv.vue";
+
+    export default {
+        components: {Csv, Editor},
+        props: {
+            content: {
+                type: String,
+                required: true
+            },
+            extension: {
+                type: String,
+                required: true
+            }
+        },
+        computed: {
+            extensionToMonacoLang() {
+                switch(this.extension) {
+                case "json":
+                    return "json";
+                case "yaml":
+                case "yml":
+                case "ion":
+                    // little hack to get ion colored with monaco
+                    return "yaml";
```

Comment:
shouldn't it be json not yaml ?

Reply:
Using JSON only returns an error as there is no quotes around properties; using YAML renders well